### PR TITLE
build(deps): switch to crates.io dependencies

### PR DESCRIPTION
**Pull request by evilrobot-01**
Switches to crates.io dependencies.

### CODE REVIEW

**Review by peterwht — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -23,124 +23,124 @@ resolver = "2"
...
-polkadot-primitives = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.1", default-features = false }
-polkadot-runtime-common = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.1", default-features = false }
-rococo-runtime-constants = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.1", default-features = false }
-rococo-runtime = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.1", default-features = false }
-xcm = { package = "staging-xcm", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.1", default-features = false }
-xcm-builder = { package = "staging-xcm-builder", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.1", default-features = false }
-xcm-executor = { package = "staging-xcm-executor", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.1", default-features = false }
+pallet-xcm = { version = "=8.0.1", default-features = false }
```
Comment (Daanvdplas) [inline Cargo.toml:111]: typo: `version = "8.0.1"`

**Review by Daanvdplas — APPROVED** [review]
Just curious how you did this? Ensuring that the crates are compatible

File: Cargo.toml (hunk 1 above)
Comment (evilrobot-01) [reply Cargo.toml:111]: It's intentional, look at the commits.

**Comment by evilrobot-01** [comment]
> Just curious how you did this? Ensuring that the crates are compatible

There are crates release branches in the SDK with version numbers, so it's done manually by looking at the versions there. Hope to update pop with some utility which automates this, at least from the manifest perspective.

`pop update parachain v1.8.0`

**Review by al3mart — APPROVED** [review]

**Comment by evilrobot-01** [comment]
No easy rebase due to squash, cherry picking into a new branch from main, will test and then push.

**Review by peterwht — APPROVED** [review]

**Pull request merged**